### PR TITLE
Add document for passing text to slots

### PR DESCRIPTION
**Pull request by 38elements**
This is related to #229 .

### CODE REVIEW

File: docs/en/api/options.md [hunk 1]
```diff
@@ -54,12 +54,33 @@ const wrapper = shallow(Component, {
...
 ```
 
+#### Passing text
+
+You can pass text to `slots`.  
+There are two limitations to this.
+
+This works at vue@2.2+.
+
+This works for the text as below.
```
Comment (eddyerburgh) [inline docs/en/api/options.md]: >This works for the text as below.

to 

>This works for the text below.

File: docs/en/api/options.md [hunk 2]
```diff
@@ -54,12 +54,33 @@ const wrapper = shallow(Component, {
...
+
+```js
+const wrapper = mount(ComponentWithSlots, { slots: { default: 'foobar' }})
+```
+
+This does not work for the text as below.
```
Comment (eddyerburgh) [inline docs/en/api/options.md]: This does not work for the text as below.

>

This does not work for the text below.

File: docs/en/api/options.md [hunk 3]
```diff
@@ -54,12 +54,33 @@ const wrapper = shallow(Component, {
...
 })
 expect(wrapper.find('div')).toBe(true)
 ```
 
+#### Passing text
+
+You can pass text to `slots`.  
+There are two limitations to this.
+
+This works at vue@2.2+.
```
Comment (eddyerburgh) [inline docs/en/api/options.md]: > This works at vue@2.2+

to

> This works with Vue 2.2+

**Comment by 38elements** [comment]
Thank you for reviewing.
I changed this.

**Pull request merged**